example/generated_examples_test.go

```diff
@@ -1606,3 +1607,52 @@ func TestTestHelpersTestClockAdvance(t *testing.T) {
 	result, _ := testhelpers_testclock.Advance("clock_xyz", params)
 	assert.NotNil(t, result)
 }
+
+func TestCustomerCreateFundingInstructions(t *testing.T) {
+	params := &stripe.CustomerCreateFundingInstructionsParams{
+		BankTransfer: &stripe.CustomerBankTransferParams{
```

Comment:
Need to fix this to include the custom method name (ie. it should be `CustomerCreateFundingInstructionsBankTransferParams`)